srcpkgs/blender/template

```diff
@@ -19,14 +19,14 @@ makedepends="libgomp-devel libpng-devel tiff-devel python3-devel glu-devel
  libopenexr-devel libopenjpeg2-devel libXi-devel openimageio-devel
  opencolorio-devel opencollada-devel python3-numpy libXrender-devel
  OpenSubdiv-devel tbb-devel libxml2-devel openvdb-devel alembic-devel
- pugixml-devel"
+ pugixml-devel libXxf86vm-devel"
 depends="desktop-file-utils hicolor-icon-theme"
 short_desc="3D graphics creation suite"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later"
```

Comment:
All the components that together make Blender are compatible under the newer GNU GPL Version 3. **That is also the license to use for any distribution of Blender binaries.**